deepfence_worker/ingesters/common.go

```diff
@@ -207,3 +207,38 @@ func anyCompleted(data []map[string]interface{}) bool {
 
 	return complete
 }
+
+func getEntityIdFromScanID(scanId, scanType string,
+	tx neo4j.Transaction) (string, error) {
+
+	entityId := ""
+	query := `MATCH (s:` + scanType + `{node_id:'` + scanId + `'}) - [:SCANNED] -> (n)
+		WITH labels(n) as label, n
+		RETURN 
+		CASE 
+    		WHEN 'ContainerImage' IN label or 'Container' in label 
+			THEN [(ci:ContainerImage{node_id:n.docker_image_id}) - [:IS] -> (cis) | cis.node_id] 
+    		WHEN 'Node' IN label THEN [n.node_id]
+    		ELSE [""]
```

Comment:
Shall we merge those two together?
Default case we just return `n.node_id`

Reply:
Done.